examples/meta/src/kernel/linear.sg

```diff
@@ -0,0 +1,21 @@
+File traindat = '../data/fm_train_real.dat'
+File testdat = '../data/fm_test_real.dat'
+
+#![create_features]
+Features feats_train=features(train_fname)
+Features feats_test=features(test_fname)
+#![create_features]
+
+#![create_kernel]
+Kernel linear_kernel=kernel("LinearKernel")
+Normalizer avg_diag_normalizer = normalizer("AvgDiagKernelNormalize")
```

Comment:
KernelNormalizer avg_diag_normalizer = AvgDiagKernelNormalize()

Reply:
``` AvgDiagKernelNormalize kernel_normalizer = AvgDiagKernelNormalize() 
    linear_kernel.set_normalizer(kernel_normalizer(1.2))
```

Reply:
check spelling plz....

Reply:
```
KernelNormalizer avg_diag_normalizer = AvgDiagKernelNormalizer()
linear_kernel.set_normalizer(avg_diag_normalizer(1.2))
```
Correct?

> KernelNormalizer avg_diag_normalizer = AvgDiagKernelNormalizer()

Reply:
@NanuSai try to compile and run it locally....?

Reply:
made the change, ran make meta_examples  
got this
```
translate.TranslationFailure: 'Failed to obtain include path for Cavg_diag_normalizer or avg_diag_normalizer or Cavg_diag_normalizer or avg_diag_normalizer'
```

Reply:
I did a complete overhaul, pulled the latest develop branch, created a new branch as it's offshoot. I would like to create a new PR and close down this one, I hope you won't mind. Really sorry for the inconvenience caused. 

I've also noticed there has been some changes in .sg file porting and is now referred as ".sg.in". I'll make sure my new PR accounts for it.